wallet/src/util/WalletBackendAdapter.ts

```diff
@@ -259,7 +259,8 @@ export const makeWalletBridgeFromFollowers = (
         },
         err => {
           console.error('error watching beansOwing', err);
-          errorHandler(new Error(err));
+          // Don't throw on beansOwing, for upgrade backwards compatibility.
+          // errorHandler(new Error(err));
```

Comment:
not clear why this line is commented out instead of deleted. we shouldn't leave commented-out code but I'm approving given the time pressure.